draft-iab-use-it-or-lose-it.md

```diff
@@ -91,17 +91,21 @@ informative:
 
 --- abstract
 
-The ability to change protocols depends on exercising the extension and version
-negotiation mechanisms that support change.  Protocols that don't use these
-mechanisms can find it difficult and costly to deploy changes.
+The ability to change protocols depends on exercising the extension
+and version negotiation mechanisms that support change.
+Implementations that don't use these mechanisms on a regular basis can
+find it difficult and costly to deploy changes.  This memo provides
+suggestions to maintain the value of evolutionary mechanisms.  The
+focus is primarily the peer-to-peer application layer.  Some lessons
+may be drawn elsewhere.
```

Comment:
I don't think that the abstract needs the caveat/disclaimer.  Keep things concise.

Reply:
ok